scipy/interpolate/fitpack2.py

```diff
@@ -1798,36 +1879,37 @@ class RectSphereBivariateSpline(SphereBivariateSpline):
 
     Parameters
     ----------
-    u : array_like
+    u : (N,) array_like
```

Comment:
This information is put after along our doc and not capitalized.

Reply:
Sorry I don't get it

Reply:
Sorry what I mean is: `array_like (n,)`
For the most up to date best practices we have, I suggest you have a look at recently added code such as `stats.bootstrap` or `stats.qmc`.

Reply:
Ic [stats.qmc](https://github.com/scipy/scipy/blob/master/scipy/stats/_qmc.py#L527-L678) and I agree but this will cover then all (N, ) in fitpack2. I will take care

Reply:
You changed it to `(n,) array_like` instead of `array_like (n,)`.